ios/Classes/AirshipInboxMessageView.swift

```diff
@@ -60,6 +65,8 @@ class AirshipInboxMessageView : NSObject, FlutterPlatformView, UANativeBridgeDel
     }
 
     private func loadMessage(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+        webviewResult = result
+        AirshipEventManager.shared.notify(loadStartedEvent)
```

Comment:
Do you need to emit a `loadFinishedEvent` in the "InvalidMessage" else block below (lines 89-93)?

Reply:
The way I see it is that loadFinished means load _successfully_ finished so it wouldn't make sense to call it in error scenarios.

Reply:
I'm concerned when I see notifications that don't always "close the loop". I can imagine an app depending on seeing `loadStarted` and `loadFinished` to indicate the message has loaded. But I also understand your desire to have `loadFinished` indicate success. Perhaps we need a `loadFailed`? Otherwise, the app has no way of knowing that except for setting an arbitrary timeout, right?

Reply:
`onLoadError` is called in the app through the FlutterResult callback:
```
webviewResult(FlutterError(code:"MessageLoadFailed",
                           message:"Unable to load message",
                           details:"Message load failed"))
```

Reply:
Oh, nice. I missed that.